db/migrate/20160510174948_add_primary_address_organisation.rb

```diff
@@ -0,0 +1,10 @@
+class AddPrimaryAddressOrganisation < ActiveRecord::Migration
```

Comment:
This migration is unnecessary - there is already a mechanism for attaching address to objects